web2/htdocs/index.html

```diff
@@ -1771,9 +1771,9 @@
             suffice:</p>
 
             <pre><code>$ uaac target <span class="hi">[[= p.uaa_endpoint ]]</span>
-$ uaac token client get admin
+            $ uaac token client get admin
 
-$ uaac client add '<span class="hi">[[= p.client_id ]]</span>' \
+            $ uaac client add '<span class="hi">[[= p.client_id ]]</span>' \
```

Comment:
These whitespace changes seem unintentional, but will cause weird issues in the web UI; can we move it back?